src/Text2Image/T2IParamTypes.cs

```diff
@@ -91,7 +91,7 @@ public enum ParamViewType
 /// <param name="Subtype">The sub-type of the type - for models, this might be eg "Stable-Diffusion".</param>
 /// <param name="ID">The raw ID of this parameter (will be set when registering).</param>
 /// <param name="SharpType">The C# datatype.</param>
-/// 
```

Comment:
what

Reply:
I think there was an empty space after the `///` and my editor must have formatted that out